engines/ags/engine/script/cc_instance.cpp

```diff
@@ -1221,7 +1221,7 @@ int ccInstance::Run(int32_t curpc) {
 					// call only supports a 32-bit value. This is fine in most cases, since
 					// methods mostly set the ptr on GlobalReturnValue, so it doesn't reach here.
 					// But just in case, throw a wobbly if it reaches here with a 64-bit pointer
-					if (fnResult._ptr > reinterpret_cast<void *>(static_cast<uintptr>(0xffffffffu)))
+					if (fnResult.full() > static_cast<intptr_t>(0xffffffffu))
 						error("Uhandled 64-bit pointer result from plugin method call");
```

Comment:
```suggestion
						error("Unhandled 64-bit pointer result from plugin method call");
```

We might as well fix that typo along the way :p